layouts/index.tsx

```diff
@@ -25,15 +36,19 @@ export const DefaultMDXWrapper = ({ children, frontMatter }: Props) => (
       </Head>
     )}
 
-    <MDXStyleWrapper>
-      <AnchorsSetter>
-        {groupCodeExamples({
-          children,
-          into: CodeExamples,
-        })}
-      </AnchorsSetter>
-    </MDXStyleWrapper>
-  </>
+    <ContentWrapper>
+      <MDXStyleWrapper>
+        <AnchorsSetter>
+          {groupCodeExamples({
+            children,
+            into: CodeExamples,
+          })}
+        </AnchorsSetter>
+      </MDXStyleWrapper>
+    </ContentWrapper>
+
+    <OnThisPage />
```

Comment:
This previously appeared in an `OnThisPageWrapper`, but that wrapper's height was causing this to scroll off the screen prematurely, so I also did some additional refactoring with that